src/test/java/com/example/animalshelter/controller/AnimalControllerIT.java

```diff
@@ -37,28 +57,21 @@ public void contextLoad() {
     }
 
     @Test
-    public void shouldReturnPositiveHttpResponseWithContent() throws Exception {
-        Animal dummyAnimal = new Animal();
-        dummyAnimal.setId(4321);
-        dummyAnimal.setAge(3);
-        dummyAnimal.setGender(AnimalGender.MALE);
-        dummyAnimal.setHealthStatus(AnimalHealthStatus.HEALTHY);
-        dummyAnimal.setName("Alex");
-        dummyAnimal.setType(AnimalType.DOG);
-
-        String asJson = new ObjectMapper().writeValueAsString(dummyAnimal);
+    public void shouldReturnPositiveHttpResponseWhenAddedAnimalSuccessfully() throws Exception {
+        String asJson = new ObjectMapper().writeValueAsString(DUMMY_ANIMAL);
```

Comment:
Let's set up final static object mapper globally for all tests in some test utils/fixtures class.

Reply:
I have extracted `ObjectMapper` to util class, also extracted some common tests dummy values to have them in one place.